owncloudApp/src/main/java/com/owncloud/android/presentation/ui/sharing/fragments/PublicShareDialogFragment.kt

```diff
@@ -283,7 +291,8 @@ class PublicShareDialogFragment : DialogFragment() {
                 publicLinkPassword,
                 publicLinkExpirationDateInMillis,
                 publicLinkPermissions,
-                publicUploadPermission
+                publicUploadPermission,
+                account?.name!!
```

Comment:
really??? from something nullable, the attribute is not null... curious

Reply:
An Android Account (see [Account.java](https://developer.android.com/reference/android/accounts/Account)) always has a name, otherwise `IllegalArgumentException` would have thrown, so is not null. But the account in this Fragment might me null.